checkout-with-aws-token/action.yml

```diff
@@ -0,0 +1,54 @@
+name: "Checkout With Token From AWS"
+description: "Checks out a repository using a bot token from AWS Parameter Store."
+
+inputs:
+  aws-access-key-id:
+    required: true
+    description: AWS Access Key ID
+  aws-region:
+    description: AWS Region
+    default: "us-gov-west-1"
+  aws-secret-access-key:
+    required: true
+    description: AWS Secret Access Key
+  github-token-parameter-store-path:
+    required: true
+    description: Path in Parameter Store for Github Token
+  path:
```

Comment:
Descriptions for the below inputs were taken directly from the [`actions/checkout`](https://github.com/actions/checkout) README. By omitting defaults (other than submodules), we let that action provide its own defaults.